program-overview.md

```diff
@@ -1,18 +0,0 @@
-### Vision :rocket: 
-Djangonaut Space is a community where open source contributing newcomers, specifically people from historically excluded groups, are welcome and supported to get on their feet to contribute to the Django project.
```

Comment:
Same, this file is not used via a link on the homepage or landing page?

Reply:
Yes this is not used 👍